keps/sig-node/4540-strict-cpu-reservation/README.md

```diff
@@ -80,16 +83,37 @@ Admission is only comparing the cpu requests against the allocatable cpus. Since
 We propose to add a new `CPUManager` policy option called `strict-cpu-reservation` to the `static` policy of `CPUManager`.
 With this policy option, we remove the reserved cores from the list of all available cores at the stage of calculation DefaultCPUSet. As a result, burstable and best-effort containers are launched with a cpuset in which the reserved cores are excluded. 
 
+### User Stories (Optional)
+
+#### Story 1
+System daemons and interrupt processing are there to serve all processes including guaranteed workloads; they needs be isolated from bursty workloads.
+
+#### Story 2
+Isolating system daemons and interrupt processing is especially critical in all-in-one deployments where workloads are placed on combined master+worker(+HCI storage) nodes.
+
 ### Risks and Mitigations
 
 The risks of adding this new feature are quite low.
-It is isolated to a specific policy option within the `CPUManager`, and is protected both by the option itself, as well as the `CPUManagerPolicyAlphaOptions` feature gate (which is disabled by default).
+
+The feature is isolated to a specific policy option within the `CPUManager`, and is protected both by the option itself, as well as the `CPUManagerPolicyAlphaOptions` feature gate (which is disabled by default).
```

Comment:
true, but the feature gate is meant to be temporary until the feature graduates. And when becomes GA, this form of protection will be lost. Which is manageable, but still.

Reply:
This is a good point!

Reply:
comment taken

Reply:
Absolutely, the feature default (default to off in our case) is independent from the GA graduation. So we can (and IMO should for the time being) keep it opt-in and safe for ppl which don't need this feature.
But at the same time, the feature gate protection is time-limited. Which is manageable (it's implicit in the feature gate lifecycle). Perhaps I'm simply too nitpicky at this stage :)